jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/StatisticsHandler.java

```diff
@@ -220,89 +183,41 @@ public double getRequestTimeStdDev()
+        return _bytesWritten.longValue();
     }
 
-    @ManagedAttribute("(in ns)")
-    public double getProcessingTimeStdDev()
+    protected class StatisticsRequest extends Request.Wrapper
     {
-        return _processTimeStats.getStdDev();
-    }
-
-    private class StatisticsRequest extends Request.Wrapper implements Connection.Listener
-    {
-        private final LongAdder _bytesRead = new LongAdder();
-        private final LongAdder _bytesWritten = new LongAdder();
-        private final long _startProcessingNanoTime;
-
         private StatisticsRequest(Request request)
         {
             super(request);
-            _startProcessingNanoTime = NanoTime.now();
+            _requestStats.increment();
             addHttpStreamWrapper(this::asHttpStream);
-
-        }
-
-        public LongAdder getBytesRead()
-        {
-            return _bytesRead;
-        }
-
-        public LongAdder getBytesWritten()
-        {
-            return _bytesWritten;
-        }
-
-        @Override
-        public void onClosed(Connection connection)
-        {
-            _connectionStats.remove(getConnectionMetaData().getId());
         }
 
         public HttpStream asHttpStream(HttpStream httpStream)
         {
             return new StatisticsHttpStream(httpStream);
         }
 
-        @Override
-        public Object getAttribute(String name)
+        /**
+         * Creating a {@link StatisticsRequest} increments the {@link #getRequests() request counter} before its gets a chance
+         * of figuring out if the request is going to be handled by the {@link StatisticsHandler#getHandler() wrapped handler}.
+         * In case the wrapped handler did not process the request, calling this method decrements the request counter to
+         * compensate for the unneeded increment.
+         */
+        protected void unProcessed()
         {
-            // return hidden attributes for requestLog
-            return switch (name)
-            {
-                // TODO class.getName + extra
-                case "o.e.j.s.h.StatsHandler.bytesRead" -> _bytesRead.longValue();
-                case "o.e.j.s.h.StatsHandler.bytesWritten" -> _bytesWritten.longValue();
-                case "o.e.j.s.h.StatsHandler.spentTime" -> spentTimeNs();
-                case "o.e.j.s.h.StatsHandler.dataReadRate" -> dataRatePerSecond(_bytesRead.longValue());
-                case "o.e.j.s.h.StatsHandler.dataWriteRate" -> dataRatePerSecond(_bytesWritten.longValue());
-                default -> super.getAttribute(name);
-            };
-        }
-
-        protected long dataRatePerSecond(long dataCount)
-        {
-            return (long)(dataCount / (spentTimeNs() / 1_000_000_000F));
-        }
-
-        private long spentTimeNs()
-        {
-            return NanoTime.since(getNanoTime());
+            _requestStats.decrement();
         }
 
         protected class StatisticsHttpStream extends HttpStream.Wrapper
```

Comment:
`public`, so subclasses have an easier access to it.

Reply:
Done.